featurebyte/app.py

```diff
@@ -9,6 +9,7 @@
 from fastapi import Depends, FastAPI, Header, Request
 from starlette.websockets import WebSocket
 
+from featurebyte._overrides.typechecked_override import custom_typechecked
```

Comment:
Add underscore to `overrides` module so that it is imported first among all featurebyte modules.